### PR TITLE
New/tag opportunity #157

### PR DESCRIPTION
**Pull request by twistedomelette**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "dayjs": "1.10.8",
     "growup-shared": "file:../shared/build",
     "joi": "17.5.0",
+    "multiselect-react-dropdown": "^2.0.21",
```
Comment (nikita-remeslov) [inline frontend/package.json]: Lock lib

Comment (nikita-remeslov) [reply frontend/package.json]: Also, check react-bootstrap multiselect

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

**Review by aleksandr-kuharenko — CHANGES_REQUESTED** [review]
store tags for all opportunities

**Review by nikita-remeslov — APPROVED** [review]

File: backend/src/services/opportunity.service.ts [hunk 2]
```diff
@@ -74,6 +78,25 @@ export const createOpportunities = async (
...
+      company: companyInstance,
+      relations: ['company'],
+    } as FindManyOptions);
+
+    const opportunityTags = tags.filter(
+      (tag) => tagsName.indexOf(tag.name) !== -1,
+    );
+
+    await opportunityInstance.tags.push(...opportunityTags);
+    await opportunityInstance.save();
```
Comment (aleksandr-kuharenko) [inline backend/src/services/opportunity.service.ts]:
```suggestion
    const tags = await tagsRepository.find({
      company: companyInstance,
      relations: ['company'],
    } as FindManyOptions);

    const opportunityTags = tags.filter(
      (tag) => tagsName.indexOf(tag.name) !== -1,
    );
    opportunitiesInstance.tags = [];
    await opportunitiesInstance.tags.push(...opportunityTags);
    await opportunitiesInstance.save();
    opportunities.push(opportunitiesInstance);

```

**Review by aleksandr-kuharenko — CHANGES_REQUESTED** [review]

**Review by aleksandr-kuharenko — APPROVED** [review]

**Pull request merged**